### PR TITLE
mitosheet: add custom url for enterprise

### PR DESCRIPTION
**Pull request by naterush**
# Description

All you need to do is to run a web server that accepts a log event and id, and you can see what Mito users are doing. For Mito Enterprise!

# Testing

Provide a URL, and then just add a print to see if it's printing the correct stuff.

# Documentation

No.

### CODE REVIEW

File: mitosheet/mitosheet/telemetry/telemetry_utils.py [hunk 1]
```diff
@@ -28,6 +30,7 @@
 
 # Write key taken from segement.com
 analytics.write_key = '6I7ptc5wcIGC4WZ0N1t0NXvvAbjRGUgX' 
+analytics
```
Comment (aarondr77) [inline mitosheet/mitosheet/telemetry/telemetry_utils.py:33]: what is this?

File: mitosheet/mitosheet/telemetry/telemetry_utils.py [hunk 2]
```diff
@@ -391,4 +394,14 @@ def log(log_event: str, params: Optional[Dict[str, Any]]=None, steps_manager: Op
...
             final_params
+        )
+
+    analytics_url = steps_manager.mito_config.get_analytics_url() if steps_manager is not None else None
+    if analytics_url is not None:
+        requests.post(
+            analytics_url,
+            json={
+                'user_id': get_user_field(UJ_STATIC_USER_ID),
+                'log_event': log_event
```
Comment (aarondr77) [inline mitosheet/mitosheet/telemetry/telemetry_utils.py:405]: Do we want to add the final_params, here?

**Review by aarondr77 — COMMENTED** [review]
A couple small comments, then LGTM.

File: mitosheet/mitosheet/telemetry/telemetry_utils.py (hunk 2 above)
Comment (naterush) [reply mitosheet/mitosheet/telemetry/telemetry_utils.py:405]: I think no, not for v1.

**Comment by naterush** [comment]
Merging when tests pass

**Pull request merged**